docs/getting-started/best-practices/installation-guidelines.md

```diff
@@ -86,16 +108,16 @@ For the remainder of this document, the assumption is you will be using Windows
 
 A high availability configuration will involve setting up:
 
-- 1 to 3 Windows servers, each with 2 cores / 4 GB of RAM with the task cap set to 10 for each server.
+- 1 to 3 Windows servers, each with 2 cores / 4 GB of RAM with the task cap set to 10-15 for each server.
```

Comment:
Should we recommend a single value (15) here instead of a range?

Reply:
Yup, I've set it to 15 and adjusted the bits below that.